openmetadata-service/src/main/java/org/openmetadata/service/resources/apps/AppResource.java

```diff
@@ -517,7 +517,11 @@ public Response scheduleApplication(
           app,
           JdbiUnitOfWorkProvider.getInstance().getHandle().getJdbi().onDemand(CollectionDAO.class),
           searchRepository);
-      Response.status(Response.Status.OK).entity("App Scheduled to Scheduler successfully.");
+      if (app.getAppType().equals(AppType.Internal)) {
+        Response.status(Response.Status.OK).entity("App Scheduled to Scheduler successfully.");
+      } else {
+        Response.status(Response.Status.OK).entity("App is External, Use Ingestion for scheduling App pipeline.");
```

Comment:
why are we returning this response here? We need to handle this by creating a ingesiton pipeline to deploy the pipeline to

Reply:
ingestion pipeline creation should be done with the ingestionPipelineResource no ? although UI takes decision on the basis of appType as external to use IngestionPipelineResource , we are just handling here in case someone hits this for external Apps

Reply:
when you install the app it should be deploy the pipeline with the default or user given schedule. User won't manage it differently

Reply:
fixed